src/util/number.ts

```diff
@@ -138,7 +138,7 @@ export function round(x: number | string, precision: number, returnStr: false):
 export function round(x: number | string, precision: number, returnStr: true): string;
 export function round(x: number | string, precision?: number, returnStr?: boolean): string | number {
     if (precision == null) {
-        precision = 10;
+        precision = ROUND_SUPPORTED_PRECISION_MAX;
```

Comment:
`ROUND_SUPPORTED_PRECISION_MAX` is a safe protection for `toFixed` methods. The actual precision issue in the JS engine is less than this.

Check this in your console
```ts
(0.1 + 0.2).toFixed(20);
```

Reply:
What do you think auto calculate precision by number decimal place length?
such as :
  if (precision == null) {
        const exp = String(x).split('e')[1];
        precision = Number(-exp) || 10;
    }`
`